src/pages/claim-page.tsx

```diff
@@ -66,6 +72,9 @@ export async function getServerSideProps({ query, locale, locales, req }) {
             claim: JSON.parse(JSON.stringify(query.claim)),
             href: req.protocol + "://" + req.get("host") + req.originalUrl,
             sitekey: query.sitekey,
+            enableCollaborativeEditor: JSON.parse(
```

Comment:
question: are you sure you need to stringify and parse? Isn't it supposed to be just a boolean?

This is needed only when we are sending objects that have cyclic dependency or reference to nodejs internals.